#Essa informação é utilizada
#somente pelo VSS-Vision, para enviar os valores RGB das cores calibradas para o VSS-Viewer, assim desenhando
#um ambiente mais realista.

uint8 r
uint8 g
uint8 b
